doc/missing-references.json

```diff
@@ -23,11 +23,11 @@
       "lib/matplotlib/colorbar.py:docstring of matplotlib.colorbar:1"
     ],
     "matplotlib.axes.Axes.patch": [
```

Comment:
why is this file getting hit?

Reply:
The doc build was complaining about inherited private members (if I remember correctly). This file is automatically generated, so there are also other missing references that have been updated unrelated to this PR: https://github.com/matplotlib/matplotlib/blob/f588d2b06e5b3c3296046d2ee9f0c13831cafe1c/doc/conf.py#L184-L185